persistence-modules/java-mongodb/README.md

```diff
@@ -1,12 +1,12 @@
-## MongoDB
-
-This module contains articles about MongoDB in Java. 
-
-### Relevant articles:
-
-- [A Guide to MongoDB with Java](http://www.baeldung.com/java-mongodb)
-- [A Simple Tagging Implementation with MongoDB](http://www.baeldung.com/mongodb-tagging)
-- [MongoDB BSON Guide](https://www.baeldung.com/mongodb-bson)
-- [Geospatial Support in MongoDB](https://www.baeldung.com/mongodb-geospatial-support)
-- [Introduction to Morphia – Java ODM for MongoDB](https://www.baeldung.com/mongodb-morphia)
-- [MongoDB Aggregations Using Java](https://www.baeldung.com/java-mongodb-aggregations)
+## MongoDB

```

Comment:
Not sure what changed here. Unless something was fixed, please revert the file.

Reply:
Hey @KevinGilmore I fixed the bad line feeds. I'm not sure if my PR is OK so do don't hesitate to tell me what's wrong ( sorry for that...)